docs/excel/custom-functions-overview.md

```diff
@@ -123,7 +123,8 @@ The following code in **customfunctions.json** specifies the metadata for the `a
     ],
     "options": {
         "cancelable": true,
-        "stream": true
+        "stream": true,
+         "volatile": false
```

Comment:
@mscharlock I think there was a change and you don't actually have to specify the volatile property. I get a warning when I try to use it that my JSON is not valid. If I remove it completely my volatile function works fine. Double-check but I don't think you need to change your JSON.